common_operations/index_append.json

```diff
@@ -0,0 +1,10 @@
+{
+    "operation": "index-append",
+    "warmup-time-period": 120,
```

Comment:
Similar suggestion as the force merge request timeout..  If you do add parameters, the README's will need to be updated.

Reply:
Similar to the request-timeout comment, I added a param for this as well.